automation/terraform/infrastructure/us-west1.tf

```diff
@@ -225,3 +225,58 @@ resource "helm_release" "west1_prometheus" {
   depends_on   = [google_container_cluster.mina_integration_west1]
   force_update = true
 }
+
+# Utilities
+
+provider kubernetes {
+  config_context = local.west1_k8s_context
+}
+
+resource "kubernetes_cron_job" "integration-testnet-cleanup" {
+  metadata {
+    name      = "integration-test-cleanup"
+    namespace = "default"
+  }
+  spec {
+    concurrency_policy            = "Replace"
+    failed_jobs_history_limit     = 5
+    schedule                      = "0 8 * * *"
```

Comment:
I think this should probably run more frequently than every `8` hours. I'm observing the int test cluster filling up pretty quickly now that we are running jobs from CI.